src/spryker/docker/image/console/root/lib/task/build.sh.twig

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function task_build()
+{
+    if [ ! -f /app/composer.json ]; then
+        task "skeleton:apply"
+    fi
+
+    task "overlay:apply"
+
+{% if @('app.mode') == 'development' %}
+    passthru "composer install --no-interaction --dev --optimize-autoloader"
+{% else %}
+    passthru "composer install --no-interaction --no-dev --optimize-autoloader"
+{% endif %}
+    passthru "vendor/bin/install -r docker -s generate"
+    passthru "vendor/bin/install -r docker -s frontend"
+    passthru "vendor/bin/install -r docker -s cache"
+    # This will affect development mode
+    passthru "composer dumpautoload -a"
```

Comment:
can we wrap this in the same stuff as the app.mode == development then just inversed?

Reply:
also can move most of this code into backend build steps rather than override and disable _base work https://github.com/inviqa/harness-base-php/blob/0.2.x/src/_base/docker/image/console/root/lib/task/build/backend.sh.twig#L11

Reply:
wilco